test/test_apps.py

```diff
@@ -580,8 +580,7 @@ def test_poisson_fem(ctx_factory):
                     J[c] * w[k] * sum(ell, dpsi(i,k,ell) * dpsi(j,k,ell))
             """,
             assumptions="nels>=1 and nbf >= 1 and nels mod 4 = 0")
-
-    print(knl)
+    logger.info("%s", knl)
```

Comment:
This is kind of a weird pattern to me. Does `logger.info(knl)` not work?

Reply:
I never actually tried that, but yeah, seems to work! :))